test/Modal.spec.js

```diff
@@ -13,16 +16,18 @@ describe('<Modal />', () => {
     init: (el, options) => {
       modalInitMock(options);
       return {
+        open: modalOpenMock,
```

Comment:
open and close are the methods returned by `init` :)

Reply:
Ah, I see... Too bad I didn't catch this one! :grimacing: